frontend/src/main/scala/bloop/engine/caches/StateCache.scala

```diff
@@ -91,6 +92,7 @@ final class StateCache(cache: ConcurrentHashMap[AbsolutePath, StateCache.CachedS
       clientSettings: Option[WorkspaceSettings]
   ): Task[State] = {
     val empty = Build(from, Nil)
+    if (from == Environment.defaultBloopDirectory) return Task.now(createState(empty))
```

Comment:
I can refactor to avoid `return` here but this was the smallest diff I could come up with. I personally use `return` sometimes in cases like this when it's undesirable to indent the rest of the method body

Reply:
I sometimes use this, but rarely so. I think it's fine in this case :+1: